### DISCUSSION TITLE
关于 db.includes() API 的使用

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#676] [Q&A] [answered]
我在尝试写一个[早上好的插件](https://github.com/Mark9804/Adachi-BOT-fork/issues/86)，但是在检查数据存在的时候遇到了一点问题。

我想检查名为 `wakeup` 的数据库中是否存在一个 `qqid` key 为用户 QQ 号的对象，但是使用以下代码总是返回 false ，但是我期望它能返回 true 。
想请教一下这种情况下应该怎么使用 `db.includes()`  API 。

```js
import db from "#utils/database";

const dbName = "wakeup";
console.log(db.get(dbName, "user"));
console.log(db.includes(dbName, "user", { qqid: msg.uid }));
```

```
[
  { qqid: 1561382166, time: 1645884526333 }
]

false
```

**Comment by Arondight** [comment, marked as answer]
![图片](https://user-images.githubusercontent.com/13673702/155847509-1d26ac13-e597-4584-8044-f2e6cb8c235c.png)

```
db.includes(dbName, "user", "qqid", msg.uid)
```

**Reply by Arondight** [reply]
我说你咋写错了，我例子给抄错了，你也跟着写错了

**Reply by mark9804** [reply]
啊我犹豫了一下，最后用了示例
<img width="466" alt="image" src="https://user-images.githubusercontent.com/9006264/155847629-963a5641-3230-41c1-a116-dc8918266a5d.png">

**Reply by Arondight** [reply]
写错了，我改一下，上班写完了跑了下没报错就提交了

**Comment by mark9804** [comment]
写完了，感谢老大哥，要是老大哥觉得够沙雕想抄过来也可以 https://github.com/Mark9804/Adachi-BOT-fork/commit/0790c95d26f4627c57aa247320e501738000aff4
<img width="350" alt="image" src="https://user-images.githubusercontent.com/9006264/155850757-48042094-c769-4430-a5a5-a3b95dfb5487.png">

**Reply by Arondight** [reply]
无意间看到了这个，现在用问答功能写个 shell 脚本就行了
